api-server.py

```diff
@@ -101,9 +118,37 @@ def main():
     elif args.coznet:
         settings.setup_coznet()
 
+    syslog_facility = None
+    if args.syslog or args.syslog_local:
```

Comment:
I discovered a bug here in testing. With my original implementation, the type of `syslog_local` was a String, so this check would pass for the string `"0"`. Now that you've (properly) fixed it to parse the parameter as an `int`, this check no longer passes since the int `0` evaluates to false. Thus, this condition and the one just below need to be `args.syslog_local is not None`.

I don't think I have commit access to the neo-python repo, so I can't push the fix (and it seems like way overkill to create a branch on my repo to create yet-another PR from).

Reply:
Oh right, got it. Updating right now for this. You could just make an external PR to the api-server branch, which would open a PR to this PR ;)